New hosts UI features (#1736)
theforeman/foreman-documentation: guides/common/modules/proc_enabling-tracer-on-a-host.adoc

> :_content-type: PROCEDURE
> 
> [id="enabling-tracer-on-a-host_{context}"]
> = Enabling Tracer on a host

Reviewer: ```suggestion
[id="enabling-tracer-on-a-host_{context}"]
= Enabling Tracer on a Host
```

> :_content-type: PROCEDURE
> 
> [id="enabling-tracer-on-a-host_{context}"]

Reviewer 2: ```suggestion
[id="enabling-tracer-on-a-host_{context}"]
```

= Enabling Tracer on a Host

Use this procedure to enable Tracer on {Project} and access Traces.
Tracer displays a list of services and applications that need to be restarted.
Traces is the output generated by Tracer in the {ProjectWebUI}.

.Prerequisites
* {project-client-name} repository is synced

> * {project-client-name} repository is avaliable in the Content View and the Lifecycle environment

Reviewer 3: ```suggestion
* {project-client-name} repository is available in the host's content view and lifecycle environment
```
